### DISCUSSION TITLE
No TX working only RX

### DISCUSSION THREAD

**Discussion by alcantir** [peterantypas/maiana#113] [Q&A]
Hi,
Everything looks ok, but I cannot get the device to transmit my position. I'm using a separate AIS receiver to verify.
- Assembled with included USB interface card
- Provided station ID (and other details)
- Get a proper GPS fix and GPS LED turn on
- Added Tx switch and Tx LED. Tx LED turn on/off when flicking switch but no transmit.
See below for boot sequence (replaced MMSI, ship name and callsign):
----
reboot
$PAISYS,11.5.0,4.0.0,,STM32L422,1,1*03
$PAISTN,123456789,MYSHIP,XX1234,36,10,4,2,10*38
$PAITXCFG,1,1,1,1,1*0B
$PMTK011,MTKGPS*08
$PMTK010,001*2E
$PMTK011,MTKGPS*08
$PMTK010,002*2D
$GPRMC,170712.341,V,,,,,0.00,0.00,210722,,,N,V*37
$GPVTG,0.00,T,,M,0.00,N,0.00,K,N*32
$GPGGA,170712.341,,,,,0,0,,,M,,M,,*4C
$GPGSA,A,1,,,,,,,,,,,,,,,,1*03
$GPGSA,A,1,,,,,,,,,,,,,,,,2*00
$GPGSV,3,1,12,09,80,206,,04,57,082,,06,49,247,,11,38,300,,0*6F
$GPGSV,3,2,12,02,31,309,,07,22,185,,03,22,125,,26,21,057,,0*66
$GPGSV,3,3,12,20,16,300,,16,15,086,,29,07,002,,31,02,033,,0*63
$GLGSV,2,1,07,70,79,318,,79,43,071,37,69,31,086,,71,29,277,30,1*7B
$GLGSV,2,2,07,85,26,231,,78,18,017,,87,15,350,,1*4B
$GPGLL,,,,,170712.341,V,N*7E
$GPRMC,170713.315,V,,,,,0.64,351.10,210722,,,N,V*33
$GPVTG,351.10,T,,M,0.64,N,1.19,K,N*3F
$GPGGA,170713.315,,,,,0,0,,,M,,M,,*4C
$GPGSA,A,1,,,,,,,,,,,,,,,,1*03
$GPGSA,A,1,,,,,,,,,,,,,,,,2*00
$GPGSV,3,1,12,09,80,206,29,04,57,082,23,06,49,247,30,11,38,300,28,0*6C
$GPGSV,3,2,12,02,31,309,,07,22,185,24,03,22,125,,26,21,057,,0*60
$GPGSV,3,3,12,20,16,300,,16,15,086,,29,07,002,,31,02,033,,0*63
$GLGSV,2,1,07,70,79,318,,79,43,071,37,69,31,086,25,71,29,277,30,1*7C
$GLGSV,2,2,07,85,26,231,,78,18,017,,87,15,350,,1*4B
$GPGLL,,,,,170713.315,V,N*7E
$GNRMC,170714.324,A,5840.0904,N,01711.2168,E,0.47,351.10,210722,,,A,V*05
$GPVTG,351.10,T,,M,0.47,N,0.88,K,A*38
$GPGGA,170714.324,5840.0904,N,01711.2168,E,1,10,1.01,4.6,M,25.7,M,,*61
$GNGSA,A,3,09,04,06,26,07,03,16,,,,,,1.31,1.01,0.83,1*05
$GNGSA,A,3,69,79,71,,,,,,,,,,1.31,1.01,0.83,2*0D
$GPGSV,4,1,13,09,80,206,30,04,57,082,31,06,49,247,29,11,38,300,27,0*66
$GPGSV,4,2,13,02,31,309,,07,22,185,25,03,22,125,32,26,21,057,32,0*67
$GPGSV,4,3,13,20,16,300,27,16,15,086,26,40,15,137,,29,07,002,,0*61
$GPGSV,4,4,13,31,02,033,,0*57
----
Any clue to what I may have missed?
//Peter

**Comment by peterantypas** [comment]
Are there any $PAITX sentences in the output? They will be infrequent when stationary.

**Reply by alcantir** [reply]
Hi Peter, thnx for responding.

Yes, I get $PAITX messages like the one below.
> $PAITX,A,18*1C

**Comment by alcantir** [comment]
Just want to add,
I have now once seen myself on the AIS receiver (!). The MAIANA was below deck and the AIS receiver antenna on top of my mast, so not optimal conditions for sure, so I took the MAIANA out and directed it against my masthead AIS (assuming beam propagation is sideways) but my boat did not appear again. 14 PAITX messages in 10 min, every second is an A message and a B message. The only information is the one below.
$PAITX,A,18*1C
$PAITX,B,18*1F
In the same period my AIS receiver was tracking 95 targets with the furthest away was 40 nm.

Hope this helps.

**Comment by peterantypas** [comment]
Is this something you built, or did you purchase a kit? What kind of antenna are you using?

**Reply by alcantir** [reply]
This is the MAIANA Base kit, so antenna is integrated.

**Comment by peterantypas** [comment]
Do you have any means by which to verify the transmitter is actually
working (spectrum analyzer, RF power meter?)

**Reply by alcantir** [reply]
I do not. All I have is the AIS receiver on the boat which does not see me.

**Reply by gah67** [reply]
It is very possible that your own receiver will not see you as it may be overloaded by the signal.
If you look at some of the local targets, is it possible to go over to them and see if they can see you?

**Reply by alcantir** [reply]
Interesting point, thnx for the suggestion.  Never thought of any overloading issue. The antennas are appx 15m away, do you think that could be causing an overload?

(Currently I'm out sailing for another two weeks and usually we moor where there are no or very few other boats, so not easy to test your suggestion out. I'll see if I can try when in a marina next time. My ship never popped up on any AIS tracker though other around me did. I know, doesn't say too much.)

**Reply by astuder** [reply]
In a quiet/low traffic area, you could also tune your VHF radio to 161.975 MHz (channel A) and 162.025 MHz (channel B). AIS transmissions should cause a very short burst of noise. If the noise correlates with the PAITX messages it would indicate that the transponder probably works.

**Comment by peterantypas** [comment]
Send an email to maiana.kits at gmail please.
